server/core/services.py

```
@@ -84,4 +86,134 @@ def logout_user(request):
+                raise ValidationError({'project': e.messages})
+            
+    def edit_club(user: User, data: dict):
+        try:
+            club = Club.objects.get(id=data.get("id"))
+            club.name = data.get('name', club.name)
+            club.description = data.get('description', club.description)
+            club.save()
+            return club
+        
+        except ValidationError as e:
+                raise ValidationError({'project': e.messages})
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)